lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -47,6 +47,20 @@ def authorize(money, payment, options = {}, capture = false)
         commit('/pts/v2/payments/', post)
       end
 
+      def void(authorization, options = {})
+        payment, amount = authorization.split('|')
```

Comment:
Is amount required for a void? I don't think we would support something like partial voids...

Reply:
Yes, original amount mandatory. 
Reference here: 
https://developer.cybersource.com/content/dam/docs/cybs/en-us/payments/developer/ctv/rest/payments.pdf 
page 52